manoseimas/mps_v2/static/components/filter_component.jsx

```diff
@@ -203,6 +227,53 @@ var SidebarFilter = React.createClass({
   }
 });
 
+var SidebarSubtabs = React.createClass({
+  componentDidMount: function() {
+    if (this.props.sticky_context){
+      $('.ui.sticky').sticky({
+        context: this.props.sticky_context,
+        offset: 70
+      });
+    }
+  },
+
+  setSelected: function(key) {
+    this.props.callback('lobbyists', key);
```

Comment:
There may be. It is consistent with code for mps_v2. I found it quite clear in their code.